tests/userTest.js

```diff
@@ -57,9 +65,18 @@ describe('Main: Currently testing user management,', function () {
       })
   }
 
-  const changeUserAttributeInteraction = function (provider, requestName, username, requestBody, response) {
+  const changeUserAttributeInteraction = async function (provider, requestName, username, requestBody, response) {
+    if (username === config.testUser) {
+      await provider
+        .given('the user is recreated', { username: username, password: config.testUserPassword })
```

Comment:
same as above, I would suggest to recreate any user that is not admin or notexisting user